Octokit/Models/Response/UserType.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Octokit
+{
+    public enum UserType
```

Comment:
/khanify api inconsistency

I think `AccountType` is the best name. So here's an idea. Make the search one `AccountSearchType` or `AccountRequestType` and keep this one `AccountType`.

What do you think? My reasoning is I bet people will see this one much more often than they'll use the search api so I want to give this one the nice name.